CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
 # Changelog
+### Release 2.2.0 (April 5, 2019)
```

Comment:
The `CHANGELOG.md` and `README.md` should have the same contents for the release notes.

Reply:
Thanks for catching that, Updated changelog, but not the README after reviews. I will sync them.